AI #2359: Add Actor (`Consumer ID` and `Principal ID`) columns to `Cost and Usage` dataset (#2360)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/conditions/includesidentityattribution.md

> # Includes Identity Attribution

Reviewer: ## TF2 Action Item - 2026-08-05

@flanakin read the presence condition aloud and confirmed the verifiable state it describes holds up: it applies when the way a data generator charges customers includes usage or purchases initiated by an account or principal. The concern @flanakin raised was the condition's name rather than its substance.

@Matt-Cowsert agreed that a condition named "includes identity attribution" leaves the reader without a referent, because no column in the set carries "identity" now that the columns are `PrincipalId`, `CredentialId`, and `CredentialDetails`. @flanakin observed that the word to change is "identity" rather than "attribution", since the condition already describes activity initiated by a principal. @Matt-Cowsert took the refinement.

**Scope:**
- [x] Rename the condition so it references principal rather than identity
- [x] Update every reference to the condition name across the PR to the new name

**Assignee:** @Matt-Cowsert

Reviewer: Both items are complete, with the naming superseded once along the way.

Commit `d0739a27` renamed the condition to `IncludesPrincipalAttribution`, which met this item as written. The open forum on 2026-08-21 then moved it again, and commit `5a04c7b4` renamed it to `IncludesRequesterAttribution` and rekeyed it on attributing usage or purchase of resources or services to `requesters`.

The reason for the second move: keyed on principals, the condition evaluated false for a data generator that publishes credential identifiers but cannot resolve a principal, so no presence requirement reached `CredentialId` on #2553 even where the generator held the value. `Requester` is the umbrella over the principal and the credential, and it carries a glossary entry.

Both renames covered the condition file, the Condition ID, the Display Name, the conditions overview row, and both references in the Cost and Usage dataset.

> The Includes Identity Attribution condition represents a verifiable state indicating whether the [*operating model*](#glossary:operating-model) includes usage or purchase of [*resources*](#glossary:resource) or [*services*](#glossary:service) initiated by a [*principal*](#glossary:principal).

Reviewer 2: ```suggestion
The Includes Identity Attribution condition represents a verifiable state indicating whether the [*operating model*](#glossary:operating-model) includes attributing usage or purchase of [*resources*](#glossary:resource) or [*services*](#glossary:service) to [*principals*](#glossary:principal).
```

> The Includes Identity Attribution condition represents a verifiable state indicating whether the [*operating model*](#glossary:operating-model) includes attributing usage or purchase of [*resources*](#glossary:resource) or [*services*](#glossary:service) to [*principals*](#glossary:principal).

Reviewer 3: 🤖 [AI][Claude] 💡 **For discussion**

The rename off `IncludesPrincipals` fixed the near-always-true problem, but "attributing usage or purchase ... to *principals*" still reads as a test of *what the provider does today* rather than *what the workload is*. A provider that could attribute per-user usage but doesn't emit it yet can read this as false — which loses the nudge.

Suggest keying on the workload, not the provider's current behavior:

```suggestion
The Includes Identity Attribution condition represents a verifiable state indicating whether the [*operating model*](#glossary:operating-model) includes usage or purchase of [*resources*](#glossary:resource) or [*services*](#glossary:service) initiated by a [*principal*](#glossary:principal).
```

This makes it a property of the operating model (does a principal initiate the usage/purchase?) rather than a property of the billing output (does the provider populate it?). A VM whose charge is associated with the VM, not a user, still reads false. A per-seat charge that names the seat holder, or per-user API/GenAI usage, reads true — even for a provider not emitting `PrincipalId` today, which is the intended nudge. Keeping the "or purchase" clause preserves the marketplace-purchaser case where the purchasing principal matters. Same shape as `IncludesRegions` (you deploy to regions → `RegionId` required), not "you happen to populate it."

Flagging for TF-2 since this shifts the gate from attributability-in-practice to principal-initiated-workload.
